x-pack/plugins/maps/public/classes/styles/color_palettes.ts

```diff
@@ -161,6 +189,31 @@ export function getOrdinalMbColorRampStops(
   );
 }
 
+// Returns an array of color stops
+// [ stop_input_1: number, stop_output_1: color, stop_input_n: number, stop_output_n: color ]
+export function getPercentilesMbColorRampStops(
+  colorPaletteId: string | null,
+  percentiles: PercentilesFieldMeta
+): Array<number | string> | null {
+  if (!colorPaletteId) {
+    return null;
+  }
+
+  const paletteObject = COLOR_PALETTES.find(({ value }: COLOR_PALETTE) => {
```

Comment:
replace with `NUMERICAL_COLOR_PALETTES` for clarity and remove `getPalette` ducktype-check?


I feel this should throw if the palette doesn't have getPalette, because this would be a bug.